recipes-apps/python-webserver/python-webserver.bb

```diff
@@ -27,12 +27,12 @@ SRC_URI = "file://cgi.py \
      file://html/usb3_results.html \
      file://images/Avnet_logo_tagline_rgb.png \
      file://images/favicon.ico \
-     file://images/board.jpg \
```

Comment:
Could be easier to just call the minized image 'board.jpg' (even if it's not really the image of the board)